metrics/pcp/pcp.go

```diff
@@ -0,0 +1,129 @@
+package pcp
+
+import (
+	"github.com/go-kit/kit/metrics"
+	"github.com/performancecopilot/speed"
+)
+
+// Reporter encapsulates a speed client
+type Reporter struct {
+	c *speed.PCPClient
+}
+
+// NewReporter creates a new reporter instance
+func NewReporter(appname string) *Reporter {
+	c, err := speed.NewPCPClient(appname)
+	if err != nil {
+		panic(err)
+	}
+	return &Reporter{c}
+}
+
+// Start starts reporting currently registered metrics to the backend
+func (r *Reporter) Start() { r.c.MustStart() }
+
+// Stop stops reporting currently registered metrics to the backend
+func (r *Reporter) Stop() { r.c.MustStop() }
+
+////////////////////////////////////////////////////////////////////////////////////////
+
+// Counter implements metrics.Counter via a single dimensional speed.Counter
+// for now, see https://github.com/performancecopilot/speed/issues/32
+type Counter struct {
+	c speed.Counter
+}
+
+// NewCounter creates a new Counter
+//
+// this requires a name parameter
+// and optionally takes a couple of string directions, which
```

Comment:
The description strings are required at the time of construction and are reported with the metric.